cl/citations/management/commands/cl_find_citations_recap.py

```diff
@@ -47,12 +60,20 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="ending creation date for a range of documents (inclusive) (filters based on when object entered into database)",
         )
         parser.add_argument(
-            "--start-date",
+            "--start-upload-date",
             type=int,
             help="starting upload date for a range of documents (inclusive) (filters based on date document uploaded to PACER)",
         )
         parser.add_argument(
-            "--end-date",
+            "--end-upload-date",
             type=int,
             help="ending upload date for a range of documents (inclusive) (filters based on date document uploaded to PACER)",
         )
+        parser.add_argument(
+            "--all",
+            type=bool,
+            help="Parse citations for all RECAP documents",
+        )
+
+    def handle(self, *args: List[str], **options: OptionsType) -> None:
+        super(Command, self).handle(*args, **options)
```

Comment:
Correct me if I'm wrong, but...this script doesn't actually *do* anything, does it?

Reply:
It does not, I just started working from the same branch while waiting you for to finish reviewing the previous stuff lol. I have a new working branch locally now. 

I am travelling atm, will make the requested changes next week when I am back

Reply:
Great. If you want to just remove this file for now, that's probably the easiest thing, I think. Sorry to take so danged long.

Reply:
I commented it out--does that work?